### PR TITLE
Issue 1872 - Antify Deposit/Withdraw Modals

### LINKED ISSUE #1872: [6] Gateway Modal ANT Implementation
1. Update UI (use style-guide, fix UX issues)
2. Test it well for all bridges/gateways
3. Remove word "Beta" from menu
4. Decide something about old withdraw page

### PR DESCRIPTION
**Pull request by startailcoon**
- Full refactor of DepositWithrawAssetSelector

Updates
- AssetSelect updated to use correct asset name replacer
- AmountSelect now works without label

<h2>General</h2>
Closes #1872 

<h2>General</h2>
lease make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at 
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [x] Check for unused code
- [x] No unrelated changes are included
- [x] None of the changed files are reformatting only
- [x] Code is self explanatory or documented
- [x] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [x] Chrome 
- [ ] Opera
- [x] Firefox
- [ ] Safari

<h2>User interface changes</h2>

_Delete this section if there weren't any UI changes. Please make sure you tested your changes in all themes_

- [x] Dark
- [x] Light
- [x] Midnight

_Please provide screenshots/licecap of your changes below_

withdrawdepositmodasmall

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
Deposit and Withdraw Modals do not open from Menu anymore

**Comment by startailcoon** [comment]
Strange, will investigate. Didn't see this issue, did you test it towards the recent change that fixed the issue with it?

**Comment by startailcoon** [comment]
https://github.com/bitshares/bitshares-ui/pull/2483

**Comment by startailcoon** [comment]
Can't reproduce any issue with opening modals from menu.

withdrawdepositmodal_menu

File: app/components/Modal/WithdrawModalNew.jsx [hunk 1]
```diff
@@ -61,6 +60,7 @@ class WithdrawModalNew extends React.Component {
             hasPoolBalance: null,
             feeError: null,
             fee_asset_id: "1.3.0",
+            fee_asset: ChainStore.getObject("1.3.0"),
```
Comment (sschiessl-bcp) [inline app/components/Modal/WithdrawModalNew.jsx]: Where is `fee_asset` used?

File: app/lib/common/assetGatewayMixin.js [hunk 2]
```diff
@@ -191,32 +177,32 @@ function gatewaySelector(args) {
...
-                            }}
-                        />
+                            {/* 
+                                NOTE 1
+                                If we have a withdraw, it would be very nice to have
+                                the value of each gateway aligned to the right so
+                                the user doesn't have to select a gateway to see the balance 
+
+                                NOTE 2
+                                This list should be sorted alphabetically to limit
```
Comment (sschiessl-bcp) [inline app/lib/common/assetGatewayMixin.js]: Why isn't it then? :)

File: app/lib/common/assetGatewayMixin.js [hunk 3]
```diff
@@ -191,32 +177,32 @@ function gatewaySelector(args) {
...
-                            title="icons.chevron_down.gateways"
-                            style={{
-                                position: "absolute",
-                                right: "10px",
-                                top: "10px"
-                            }}
-                        />
+                            {/* 
+                                NOTE 1
+                                If we have a withdraw, it would be very nice to have
```
Comment (sschiessl-bcp) [inline app/lib/common/assetGatewayMixin.js]: Can you add a screen what you mean here please?

Comment (sschiessl-bcp) [reply app/lib/common/assetGatewayMixin.js]: How was this resolved?

File: app/lib/common/assetGatewayMixin.js (hunk 2 above)
Comment (sschiessl-bcp) [reply app/lib/common/assetGatewayMixin.js]: Did you add sorting, or why is this resolved?

**Comment by startailcoon** [comment]
Sorry, must have missed posting my comments when resolving these. Both these notes are fixed as part of the following PR https://github.com/bitshares/bitshares-ui/pull/2523.

**Comment by sschiessl-bcp** [comment]
Please resolve the conflict, I'm unsure which is the newest
image

**Comment by startailcoon** [comment]
Resolved @sschiessl-bcp

**Pull request merged**